pkgs/applications/audio/lsp-plugins/default.nix

```diff
@@ -154,6 +142,5 @@ stdenv.mkDerivation rec {
       homepage = https://lsp-plug.in;
       maintainers = with maintainers; [ magnetophon ];
       license = licenses.gpl2;
-      platforms = [ "x86_64-linux" ];
```

Comment:
Can you use `platforms.linux` if it does not build on Darwin and `platforms.unix` if it does?

Reply:
I don't know if it does. Let's find out!

Reply:
Does not seem to build on Darwin https://logs.nix.ci/?key=nixos/nixpkgs.76317&attempt_id=39ed18c1-8ad6-47d2-a68b-e16a5fecab6d

Reply:
OK, changed it to linux.